test/SourceKit/CodeComplete/complete_sort_order.swift

```diff
@@ -33,6 +33,8 @@ func test() {
 // CONTEXT: key.kind: source.lang.swift.decl
 // CONTEXT-NEXT: key.name: "x"
 // CONTEXT-NOT: key.name:
+// CONTEXT: key.name: "complete_sort_order",
+// CONTEXT-NOT: key.name:
 // CONTEXT: key.name: "foo(a:)"
 // CONTEXT-NOT: key.name:
 // CONTEXT: key.name: "foo(a:)"
```

Comment:
`complete_sort_order` is the module name of the current file. It's a little strange if it precedes other decls in the module.
It probably makes sense if `Decl[Module]` has lower priority than other decls.
@benlangmuir what do you think?

Reply:
I agree